GoMeet/src/main/java/com/google/sps/servlets/LocationServlet.java

```diff
@@ -1,38 +1,37 @@
 package main.java.com.google.sps.servlets;
 
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.Key;
-import com.google.appengine.api.datastore.KeyFactory;
+import main.java.com.google.sps.data.Dao;
 import main.java.com.google.sps.data.Location;
+import main.java.com.google.sps.data.LocationDao;
+
 import com.google.gson.Gson;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.jsoup.Jsoup;
 import org.jsoup.safety.Whitelist;
 
+import com.google.sps.data.ServletUtil;
+
 /** Handles fetching and saving location data. */
 @WebServlet("/location-data")
 public class LocationServlet extends HttpServlet {
 
   private static final int INITIAL_VOTE_COUNT = 1;
-
+  private Dao<Location> locationDao = new LocationDao();
+  
  /** Responds with a JSON array containing location data. */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
 
-    Collection<Location> locations = getLocations();
-    Gson gson = new Gson();
-    String json = gson.toJson(locations);
+    List<Location> locations = locationDao.getAll();
+    String json = ServletUtil.convertToJson(locations);
```

Comment:
Double check if this is actually converting to JSON properly. I know Anna and I ran into issues with calling this function and it was returning null instead of the JSON string.

Reply:
Yup, will do. Thanks for letting me know.